packages/nimble-components/src/wafer-map/modules/matrix-renderer.ts

```diff
@@ -0,0 +1,17 @@
+import * as Comlink from 'comlink';
```

Comment:
Is the folder meant to be called 'modules', or was it supposed to be 'models'? If you intended "modules" what's the reasoning for that name?

Reply:
![image](https://github.com/ni/nimble/assets/110180309/f4a16dac-6f13-4fb8-ac00-334900be2976)
I don't have a strong point, it was there before, I assumed that files related with the wafermap should stay there. Should I put this file in a new directory src/wafer-map/models?

Reply:
I think the folder should just be re-named 'models', but that could be done in a separate PR.